compiler/repl-service/server/src/main/scala/com/digitalasset/daml/lf/ReplServiceMain.scala

```diff
@@ -216,7 +216,9 @@ class ReplService(
       ApplicationId("daml repl"),
       TimeProvider.UTC)
     runner.runWithClients(clients).onComplete {
-      case Failure(e) => respObs.onError(e)
+      case Failure(e) =>
+        System.err.println(s"$e")
+        respObs.onError(e)
```

Comment:
The REPL server was not printing errors such as when trying to re-allocate the same party. I'm working on a test-case for a separate PR.